source/whats-new.txt

```diff
@@ -35,6 +36,26 @@ Learn what's new in:
 * :ref:`Version 6.1 <version-6.1>`
 * :ref:`Version 6.0 <version-6.0>`
 
+.. _version-6.15:
+
+What's New in 6.15
+------------------
+
+The {+driver-short+} v6.15 release includes the following features:
+
+- Adds support for supplying custom AWS credentials when configuring :ref:`authentication
+  <node-authentication-mechanisms>` or :ref:`In-Use Encryption <node-fle>`. You
+  must supply the custom function signature in the form of ``() => Promise<AWS.Credentials>``.
+  You can also specify provider chains from the AWS SDK, allowing you to
+  customize your options. To view examples of how to provide custom
+  credentials, see the :github:`v6.15.0 Release Notes
+  <mongodb/node-mongodb-native/releases/tag/v6.15.0>` on GitHub.
+- Fixes bugs that caused unhandled internal testing rejections.
```

Comment:
S: Can you make it more clear how this might affect the end user? It looks likethere might have been errors being returned to the user according to the release notes. You might want to change this language from suggesting the change is internal only